independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -35,6 +35,14 @@
  */
 public class QuarkusClassLoader extends ClassLoader implements Closeable {
     private static final Logger log = Logger.getLogger(QuarkusClassLoader.class);
+    private static final Logger lifecycleLog = Logger.getLogger(QuarkusClassLoader.class.getName() + ".lifecycle");
+    private static final boolean LOG_ACCESS_TO_CLOSED_CLASS_LOADERS = Boolean
+            .getBoolean("quarkus-log-access-to-closed-class-loaders");
+
+    private static final short STATUS_OPEN = 1;
+    private static final short STATUS_CLOSING = 0;
+    private static final short STATUS_CLOSED = -1;
```

Comment:
Use `byte`, which is smaller than `short` 😉 
```suggestion
    private static final byte STATUS_OPEN = 1;
    private static final byte STATUS_CLOSING = 0;
    private static final byte STATUS_CLOSED = -1;
```

Reply:
See my comment above to Alexey. Will the `byte` be readable when debugging? Because I don't want cryptic values that nobody can understand when debugging already hairy issues.

Reply:
Yes, it should. I don't understand how that would be cryptic as it's a single byte, do you have any example that makes that unreadable?

Reply:
I will change it but really the amount of nitpicking is a bit annoying.

These objects are several MB large so 1 or 2 bytes are really negligible...